discover/resources/index.md

```diff
@@ -34,9 +34,9 @@ ArcGIS Pro seems to load an appropriate transformation, but you may need to set
 
 Stated horizontal accuracy of the `Google` imagery is expected to achieve or exceed one meter (CE90) in most areas without significant vertical relief. You can generally expect higher precision in urban areas where existing supplemental ground control was more abundant.
 
-The 15cm and 30cm `Hexagon` imagery have stated horizontal accuracies of 1.07 and 2.6 meters (CE90), respectively.
+The `Hexagon` imagery had stated horizontal accuracies of 1.07 and 2.6 meters (CE90), respectively.
```

Comment:
**nitpick**: consistency

```suggestion
The `Hexagon` imagery have a stated horizontal accuracies of 1.07 and 2.6 meters (CE90), respectively.
```

Reply:
i meant to type _has_ but this will work